client/ayon_blender/api/lib.py

```diff
@@ -252,19 +252,24 @@ def lsattrs(attrs: Dict) -> List:
             continue
         for node in getattr(bpy.data, coll):
             for attr, value in attrs.items():
-                avalon_prop = node.get(pipeline.AVALON_PROPERTY)
-                if not avalon_prop:
+                ayon_prop = node.get(pipeline.AVALON_PROPERTY) or (
+                    node.get(pipeline.AYON_PROPERTY)
+                )
```

Comment:
I think your reversed this the wrong way, no? I'm not sure how you interpreted my comment this way.

I'd say **AYON** property takes precedence over _any_ backwards compatibility.
As such, if there's any AYON property - that should be read first.